netbox/account/views.py

```diff
@@ -44,10 +44,19 @@ def dispatch(self, *args, **kwargs):
         return super().dispatch(*args, **kwargs)
 
     def gen_auth_data(self, name, url, params):
-        display_name, icon_name = get_auth_backend_display(name)
+        display_name, icon_source = get_auth_backend_display(name)
+
+        icon_name = None
+        icon_img = None
+        if '://' in icon_source:
```

Comment:
This will raise an exception if `icon_source` is `None`. It should probably be:
```
if icon_source and '://' in icon_source:
```

Reply:
Good catch, indeed. Patched in my amended commit.